### ISSUE TITLE
HELP: Training conditional generation on my own dataset

### ISSUE THREAD

**Issue by Walleeeda** [lucidrains/DALLE2-pytorch#308]
We are studying the inverse design of stochastic honeycomb structures. That is , we hope to feed the model a targeted property curve of a structure (i.e. the stress strain curve, concept in mechanic field).  And we want to first train a unconditional model for ease. However, we didn't get a nice result after trying hard. and I hope to get some suggestions from you.
The dataset are like below, the scatter with color represent the nucleus of the cells in the structures, and the color represent the cell wall thickness. 
![dataset_pictures](https://github.com/lucidrains/DALLE2-pytorch/assets/109721998/8b521f35-be8d-4d75-9ed4-2fb6bf6bbd18)

We trained 2000 imaged for unconditional generation. The image size is 64*64. I have tried batchsize=32 and batchsize=16, we also tried the mse loss and mae loss . After training for 2000 epochs, the result is still bad, and the mse loss is fluctuated around 0.0003. The training result is below
 training_result_using mse loss:
![training_result_mseloss](https://github.com/lucidrains/DALLE2-pytorch/assets/109721998/95f4dc2b-a1cd-421c-af54-a516697bf80e)
 
training_result_using mae loss:
![mae](https://github.com/lucidrains/DALLE2-pytorch/assets/109721998/0ad83ca6-8564-4a3d-b6ee-0460af1e31e5)

I dont know how to deal with it , hoping to get some suggestion from you!!!  Very thanks for your contribution.
  @

**Comment by Walleeeda** [comment]
Hope anyone can offer some help
